scripts/editor/output-css-variables.js

```diff
@@ -117,49 +118,61 @@ export const outputCssVariables = (attributes, manifest, unique, globalManifest)
 	// Check if component or block.
 	let name = manifest['componentClass'] ?? attributes['blockClass'];
 
-	// Iterate each variable.
-	for (const [variableName, variableValue] of Object.entries(variables)) {
 
-		// Constant for attributes set value (in db or default).
-		const attributeValue = attributes[variableName];
+	// Setting defined variables to each breakpoint.
+	const setVariablesToBreakpoints = (variables) => {
+		// Iterate each variable.
+		for (const [variableName, variableValue] of Object.entries(variables)) {
 
-		// Set internal breakpoints variable.
-		let internalBreakpoints = [];
+			// Constant for attributes set value (in db or default).
+			const attributeValue = attributes[variableName];
 
-		// If type default or value.
-		if (!Array.isArray(variableValue)) {
-			internalBreakpoints = variableValue[attributeValue] ?? [];
-		} else {
-			internalBreakpoints = variableValue;
-		}
+			// Set internal breakpoints variable.
+			let internalBreakpoints = [];
 
-		// Iterate variable array to check breakpoints.
-		internalBreakpoints.forEach((breakpointItem) => {
+			// If type default or value.
+			if (!Array.isArray(variableValue)) {
+				internalBreakpoints = variableValue[attributeValue] ?? [];
+			} else {
+				internalBreakpoints = variableValue;
+			}
 
-			// Define variables from breakpointItem.
-			const {
-				breakpoint = 'default', // If breakpoint is not set use default name
-				inverse = false, // If inverse is not set use mobile first.
-				variable = [],
-			} = breakpointItem;
+			// Iterate variable array to check breakpoints.
+			internalBreakpoints.forEach((breakpointItem) => {
 
-			// Check if we are using mobile or desktop first. Mobile first is the default.
-			const type = inverse ? 'max' : 'min';
+				// Define variables from breakpointItem.
+				const {
+					breakpoint = 'default', // If breakpoint is not set use default name
+					inverse = false, // If inverse is not set use mobile first.
+					variable = [],
+				} = breakpointItem;
 
-			// Iterate each data array to find the correct breakpoint.
-			data.some((item, index) => {
+				// Check if we are using mobile or desktop first. Mobile first is the default.
+				const type = inverse ? 'max' : 'min';
 
-				// Check if breakpoint and type match.
-				if (item.name === breakpoint && item.type === type) {
+				// Iterate each data array to find the correct breakpoint.
+				data.some((item, index) => {
```

Comment:
Shouldn't this be `map` or `forEach`?

Reply:
if I'm not wrong. This should be an optimization because it will exit as soon as the variable is inserted. 
If map/forEach was/were used it would continue through the whole array. 😄 

This might be removed/refactored when #359 is done.

Reply:
That's exactly right!